R/load_secuTrial_export.R

```diff
@@ -117,3 +118,46 @@ load_secuTrial_export <- function(data_dir, tables = "all", add_id_name = NULL)
   class(return_list) <- "secuTrialdata"
   return(return_list)
 }
+
+
+
+#' @rdname secuTrialdata
+#' @param x secuTrialdata object as returned by \code{load_secuTrial_export}
+#'
+#' @return data.frame with a row for each table in the export. For each table it
+#'         contains the name, number of rows and columns, an indicator for
+#'         whether the table is a metadata table and the files original name.
+#' @export
+#' @usage print(x)
+#'
+#' @examples
+#' # Print method
+#' print(sT_export)
+#' # or
+#' sT_export
+
+print.secuTrialdata <- function(x){
+
+  cat(paste("SecuTrial data imported from", x$export_options$data_dir, "\n"))
+
+  tab <- lapply(x$export_options$data_names, function(y){
+    tmp <- x[[y]]
+    tmp
+    data.frame(table = y,
+               nrow = nrow(tmp),
+               ncol = ncol(tmp),
+               meta = y %in% x$export_options$meta_names)
+  })
+  tab <- do.call("rbind", tab)
+  tab$original_name <- rownames(tab)
+  rownames(tab) <- NULL
+  print(tab, row.names = FALSE)
+
+}
+
+#' @rdname secuTrialdata
+#' @usage plot(x)
+plot.secuTrialdata <- function(x){
+  vs <- visitstructure(x)
```

Comment:
should this be visit_structure?

Reply:
No, it's the plot method for the secuTrialdata class. There is also a plot method for secuTrialvisit objects.

Reply:
ah...you mean line 161.... yes i think it probably should... well spotted